src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -50,6 +51,14 @@ public ScriptHostConfiguration()
         /// The default is false. When set to true, logs will be written to the directory
         /// specified by <see cref="RootLogPath"/>.
         /// </summary>
-        public bool FileLoggingEnabled { get; set; }    
+        public bool FileLoggingEnabled { get; set; }
+
+        /// <summary>
+        /// Gets the list of functions that should be run. This list can be used to filter
+        /// the set of functions that will be enabled - it can be a subset of the actual
+        /// function directories. When left null (the default) all discovered functions will
+        /// be run.
+        /// </summary>
+        public Collection<string> Functions { get; set; }
```

Comment:
Adding this as a simple way to scope the set of functions that are active. You can now add a `functions: [ "foo", "bar" ]` property to host.json to restrict the set of running functions. This is meant primarily for testing. It helped me in the work for this PR, since I could isolate a single function w/o the noise of all the others running.